qpc/insights/login_config.py

```diff
@@ -0,0 +1,45 @@
+"""AddLoginCommand is used to save insights login configuration."""
+
+from __future__ import print_function
```

Comment:
we can bury this import next to python 2 tombstone :smile: 

(print was a statement on python 2. this import was required to make it work as a function, emulating python3 behavior)